common-money.lic

```diff
@@ -25,7 +25,7 @@ module DRCM
     return (amount.to_i * 1000) if 'gold' =~ /^#{denomination}/
     return (amount.to_i * 100) if 'silver' =~ /^#{denomination}/
     return (amount.to_i * 10) if 'bronze' =~ /^#{denomination}/
-    amount
+    amount.to_i
```

Comment:
💡 In hindsight (i.e. after I already merged this) I wonder if we should instead do `amount = amount.to_i` as the first thing in the function... oh well

Reply:
Thanks for the quick review and merge. I also thought about that but not as familiar with the why it was done the way it was so chose path of least resistance 😅